flask_monitoringdashboard/core/plot/plots.py

```diff
@@ -66,12 +68,4 @@ def get_average_bubble_size(data):
     :param data: a list with lists: [[a, b, c], [d, e, f]]
     :return: a constant for the bubble size
     """
-    def get_max(my_list):
-        m = None
-        for item in my_list:
-            if isinstance(item, list):
-                item = get_max(item)
-            if not m or m < item:
-                m = item
-        return m
-    return get_max(data) / BUBBLE_SIZE_RATIO
+    return max([max([math.sqrt(r) for r in row]) for row in data]) / BUBBLE_SIZE_RATIO
```

Comment:
It's better to first find the max and then compute the sqrt, instead of computing the sqrt's of every element in the list.